cg/cli/workflow/nf_analysis.py

```diff
@@ -209,3 +209,24 @@ def store(context: click.Context, case_id: str, dry_run: bool) -> None:
     except Exception as error:
         LOG.error(repr(error))
         raise click.Abort()
+
+
+@click.command("store-available")
+@OPTION_DRY
+@click.pass_context
+def store_available(context: click.Context, dry_run: bool) -> None:
+    """Store bundles for all finished nf-analyses in Housekeeper."""
+
+    analysis_api: NfAnalysisAPI = context.obj.meta_apis[MetaApis.ANALYSIS_API]
+
+    exit_code: int = EXIT_SUCCESS
+
+    for case_obj in set([*analysis_api.get_cases_to_qc(), *analysis_api.get_cases_to_store()]):
```

Comment:
I wonder if we should add a new defintion for `get_cases_to_store` into NfAnalysisAPI so we can use the method directly without using sets of lists

Reply:
I agree! And you can make it directly a set:
```suggestion
    for case_obj in {*analysis_api.get_cases_to_qc(), *analysis_api.get_cases_to_store()}:
```

Reply:
@fevac Do you mean something like:

```
    def get_cases_to_store(self) -> list[Case]:
        """Return cases where analysis finished successfully,
        and is ready to be stored in Housekeeper."""
        return [
            case
            for case in self.status_db.get_running_cases_in_workflow(workflow=self.workflow)
            if self.trailblazer_api.is_latest_analysis_completed(case_id=case.internal_id)
            or self.trailblazer_api.is_latest_analysis_qc(case_id=case.internal_id)
        ]

```

Reply:
yes!

Reply:
Added now.